src/screens/expandedAssetSheet/components/SheetContent.tsx

```diff
@@ -89,6 +89,9 @@ export function SheetContent() {
             {/* {isOwnedAsset && (
               <CollapsibleSection content={<BridgeSection />} icon="􁾫" id={SectionId.BRIDGE} primaryText="Bridge" secondaryText={'to'} />
             )} */}
+            {tokenInteractions.length > 0 && (
```

Comment:
maybe we can use `isOwnedAsset` here but then if they sold the asset they wouldn't see this section. Not sure what else we can use to delineate this other than moving the interaction data into the context.